### ISSUE TITLE
cannot get the output segmentation in the paper with the provided weight file

### ISSUE THREAD

**Issue by opnumten** [CovertLab/DeepCell#6]
Hi,

with the provided weight file, I cannot get the output images shown in your PLOS paper. The prediction accuracy is pretty low.

Thanks

**Comment by vanvalen** [comment]
I'll take a look at it today!

Dave

David Van Valen MD, PhD
Postdoctoral Scholar, Covert Lab
Stanford University

**Comment by xinrzhsh24** [comment]
Hi, Valen~
I can get the pretty segmentation results using your trained model ( '2016-07-12_nuclei_all_61x61_bn_feature_net_61x61_*'). But the model trained by using your code of 'training_template',  the result is too bad. I also used your data (nulei_all_61*61) and the model bn_feature_net_61*61. I noticed your trained model is 1.9M, while mine is 3.8M. What is the problems?

**Comment by hftsai** [comment]
I'm seeing similar..i'm using just the trained network provided and the validation data provided (HeLa)
The Jaccard index and dice index provided in the original jupyter notebook suggest:
Jaccard index is 0.84462 +/- 0.0107597496164
Dice index is 0.914696973088 +/- 0.00655002244584

But when i run it on the docker i pulled
the indexes are only:
Jaccard index is 0.700539 +/- 0.144342
Dice index is 0.814186574197 +/- 0.115389352237b

does anyone know why this would happen?

**Comment by vanvalen** [comment]
I'm taking a look at it.

Dave

David Van Valen MD, PhD
Postdoctoral Scholar, Covert Lab
Stanford University

**Comment by hftsai** [comment]
Hi Dave, Thank you so much for your time! I'm sure it'll be extremely useful for my work if we can take a ride on your platform.

**Comment by vanvalen** [comment]
I'm not quite sure what the issue is, but the fastest fix is to train new
neural networks for the HeLa cells. They should be ready in a day or so.

David Van Valen MD, PhD
Postdoctoral Scholar, Covert Lab
Stanford University
